src/platform/datapath_xplat.c

```diff
@@ -0,0 +1,419 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Datapath Implementation (User Mode)
+
+--*/
+
+#include "platform_internal.h"
+
+#ifdef QUIC_CLOG
+#include "datapath_xplat.c.clog.h"
+#endif
+
+#pragma warning(disable:4116) // unnamed type definition in parentheses
+
+#pragma warning(disable:4100) // unreferenced
+#pragma warning(disable:6101) // uninitialized
+#pragma warning(disable:6386) // buffer overrun
```

Comment:
I rather fix these instead of ignore them. Can you please remove this and fix any issues?